docs/configure.md

```diff
@@ -37,7 +39,8 @@ You can place your temporary config in `.cache/credentials.json` as this is igno
       "clientId": "",
       "clientSecret": "xxxxxxxxxxxxxxxxxxxxxxxxx-xxxxxxxxxxx-xxxxx"
     },
-    "tenant": "mytenant"
+    "tenant": "mytenant",
+    "tld": "eu"
```

Comment:
Can you revert the changes for the code sample since it's already included in the linked docs? Appreciate it!